Source/Components/RViewModel.swift

```diff
@@ -42,19 +42,19 @@ extension RViewModel {
                 fit: fit,
                 alignment: alignment,
                 autoplay: autoplay,
-                artboard: artboard,
-                animation: animation,
+                artboard: artboardName,
+                animation: animationName,
                 stateMachine: stateMachineName
             )
         }
-        else if let httpUrl = url {
+        else if let httpUrl = webURL {
```

Comment:
Agreed. I wanted a more generic name prefix than `http` but the `URL` type can refer to local locations, so to make things more clear I went with `webURL` rather than `url`. I'm changing local vars and param labels to match wherever I find them